packages/keystone/test.utils.js

```diff
@@ -521,11 +521,11 @@ const expectToThrowAccessDeniedError = async (testFunc, path) => {
     await catchErrorFrom(testFunc, (caught) => {
         expect(caught).toMatchObject({
             name: 'TestClientResponseError',
-            data: { [path]: null },
+            data: set({}, path, null),
```

Comment:
Field accesses or what?)

Reply:
`path` can be nested, as in the case of counting objects. 
So we had to add a little more variability :)

`obj` | `objs` | `meta.count`